src/core_ocean/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -1740,6 +1747,58 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
+                           call mpas_pool_get_array(meshPool, 'latCell', latCell)
+                           call mpas_pool_get_array(meshPool, 'lonCell', lonCell)
+                           if (config_reset_debugTracers_near_surface) then
+                              !$omp do schedule(runtime)
+                              do iCell = 1, nCells
+
+                                ! Reset tracer1 to 2 in top n layers
+                                do k = 1, config_reset_debugTracers_top_nLayers
+                                   tracersGroupNew(1,k,iCell) = 2.0_RKIND
+                                end do
+
+                                ! Reset tracer2 to 2 in top n layers
+                                ! in zonal bands, and 1 outside
+                                lat = latCell(iCell)*180./3.1415
+                                if (     lat>-60.0.and.lat<-55.0 &
+                                     .or.lat>-40.0.and.lat<-35.0 &
+                                     .or.lat>- 2.5.and.lat<  2.5 &
+                                     .or.lat> 35.0.and.lat< 40.0 &
+                                     .or.lat> 55.0.and.lat< 60.0 ) then
+                                    do k = 1, config_reset_debugTracers_top_nLayers
+                                       tracersGroupNew(2,k,iCell) = 2.0_RKIND
+                                    end do
+                                else
+                                    do k = 1, config_reset_debugTracers_top_nLayers
+                                       tracersGroupNew(2,k,iCell) = 1.0_RKIND
+                                    end do
+                                end if
+
+                                ! Reset tracer3 to 2 in top n layers
+                                ! in zonal bands, and 1 outside
+                                lat = latCell(iCell)*180./3.1415
+                                if (     lat>-55.0.and.lat<-50.0 &
+                                     .or.lat>-35.0.and.lat<-30.0 &
+                                     .or.lat>-15.0.and.lat<-10.0 &
+                                     .or.lat> 10.0.and.lat< 15.0 &
+                                     .or.lat> 30.0.and.lat< 35.0 &
+                                     .or.lat> 50.0.and.lat< 55.0 ) then
+                                    do k = 1, config_reset_debugTracers_top_nLayers
+                                       tracersGroupNew(3,k,iCell) = 2.0_RKIND
+                                    end do
+                                else
+                                    do k = 1, config_reset_debugTracers_top_nLayers
+                                       tracersGroupNew(3,k,iCell) = 1.0_RKIND
+                                    end do
+                                end if
+                              end do
+                              !$omp end do
+                           end if
+                        end if
+
```

Comment:
@mark-petersen do we really want to merge these debug tracer changes into ocean/develop?

Reply:
I would like to merge in the reset of debug tracers now, because we will be conducting tests in the next few weeks where this makes it very easy to control with flags. After we are satisfied, we can remove this section when we turn on Redi mixing by default, with the tuned coefficients.  Also, with this merge the debug reset will be on the trunk once, so it is easy to find later.

Reply:
Good point.  I agree this would be good for testing.